scripts/script.js

```diff
@@ -0,0 +1,53 @@
+ 
+function cost() {
+    alert("$123.45");
+}
+

```

Comment:
You are on the right track here, but not quite what we're looking for. Remember, we want to show the cart total. So if you click two dogs, it should display $246.90". You can do this by creating a variable to keep track of cost.

Reply:
Also, you'll want to use function parameters. So something like: `func total(cost, dog, breed)` then when you call the function in your HTML code, you'd pass in the values.

Reply:
Thank you so much Tyler, I have already pulled the project again with the committed changes.